web_api/national_park/national_parks.py

```diff
@@ -21,22 +21,21 @@ def get_parks_data(query):
         return None, ex # If an exceptions occur, return None instead of data, and the exception
 
 
-def create_park_objects_list(data):
+def create_park_objects_list(park_data):
     """Create a Park object out of each object in the API response containing the park's
     name, description, state_code, location, park_code, phone, and email. 
     The function returns a list of park objects."""
-    park_data, error = data # Unpack tuple that holds either data or exception
     parks = [] # Store park objects
     
     for park in park_data:  # Create a park object for each set of results and add to the park list 
-        name = park['fullName'],
-        description = park['description'],
-        state_code = park['states'],
-        latitude = park['latitude']
-        longitude = park['longitude']
-        park_code = park['parkCode'],
-        phone = park['contacts']['phoneNumbers'][0]['phoneNumber'],
-        email = park['contacts']['emailAddresses'][0]['emailAddress']
+        name = park.get('fullName')
```

Comment:
This is getting data from a list, .get() doesn't work for this. Just curious why you changed it - wasn't it working before?

Reply:
it was working fine when i did it... 
ill retest...

Reply:
i mean it does not need to be that way.. we will just need to deal with errors instead of using the .get... and we need to remove the comas... that was the main thing

Reply:
wait no.... its a dictionary... so there should be no issues... fullName is a key

Reply:
did some more testing and it works fine...